srcpkgs/pylint/template

```diff
@@ -1,49 +1,38 @@
 # Template file for 'pylint'
 pkgname=pylint
-reverts="2.3.1_1"
-version=1.9.5
+version=2.4.4
 revision=1
 archs=noarch
 wrksrc="pylint-${version}"
-build_style=python2-module
+build_style=python3-module
 pycompile_module="pylint"
-hostmakedepends="python-setuptools python3-setuptools"
-depends="python-setuptools python-astroid python-six python-isort python-mccabe
- python-backports.configparser python-backports.functools_lru_cache
- python-singledispatch"
-short_desc="Python2 code static checker"
+hostmakedepends="python3-setuptools"
+depends="python3-setuptools python3-astroid python3-six python3-isort
```

Comment:
Does it actually need setuptools to run?

Reply:
Right, it doesn't.